pkg/webhook/mutation/pod/handler/otlp/config.go

```diff
@@ -7,3 +7,15 @@ import (
 var (
 	log = logd.Get().WithName("pod-mutation-otlp")
 )
+
+const (
+	OTLPAnnotationPrefix = "otlp-exporter-configuration"
+	// AnnotationOTLPInjectionEnabled controls whether the automatic injection of OTLP env vars and resource attributes should happen for a pod
+	AnnotationOTLPInjectionEnabled = OTLPAnnotationPrefix + ".dynatrace.com/inject"
```

Comment:
this isn't used. there is another constant with the same name in pkg/webhook/mutation/pod/mutator/config.go which is used though, along with another set of all of these constants. please only use one set